component/badge/badge.html

```diff
@@ -39,12 +39,13 @@
         <p class="sidebar-heading">Components</p>
         <ul class="sidebar-list">
           <li><a href="/component/avatar/avatar.html">Avatars</a></li>
-          <li><a href="/component/badge/badge.html">Badges</a></li>
+          <li><a href="/component/badge/badge.html" class="active">Badges</a></li>
```

Comment:
It's good you have made a single class and used where applicable

Reply:
thanks